components/TableSection/TableSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useState, useEffect } from "react";
+import styles from "./TableSection.module.css";
+
+import TableCell from "../TableCell/TableCell";
+
+interface TableSectionPropType {
+  tableData: any;
+  currentState: string;
+}
+
+const TableSection: FC<TableSectionPropType> = ({
```

Comment:
suggestion: Why did you name this `<TableSection>`? As it's a table, can be just `<Table>` :)

Reply:
I am afraid of repeat naming issues with official table of HTML5

Reply:
You won't never repeat the name because in HTML elements are lowercase `<table>`. In React, components must be CamelCase. So `Table !== table`